### PR TITLE
cypress integration and some tests created

### PR DESCRIPTION
**Pull request by LBHMVillalobos**
**What?**  
- Cypress integration 
- Cypress-dotenv plugin installed
- Tests created:
      - search by mosaic id, children name or adult name logged as admin dev
      - search by children name or adult name logged as children group
      - search by children name or adult name logged as adult group

**How to test it?**  
Testing variables need to be added to .env
yarn dev
yarn run cypress open
Click on each test to open the test on the browser

**Next?** 
- Create more tests

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -8,7 +8,7 @@
   },
   "plugins": ["react", "jest"],
   "parser": "babel-eslint",
-  "extends": ["eslint:recommended", "plugin:prettier/recommended"],
+  "extends": ["eslint:recommended", "plugin:prettier/recommended", "plugin:cypress/recommended"],
```
Comment (sirLisko) [inline .eslintrc:11]: nice one 👍

File: cypress/fixtures/example.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (sirLisko) [inline cypress/fixtures/example.json:1]: I think we can delete this file :)

File: debug.log [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[1110/150227.182:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (sirLisko) [inline debug.log:1]: We should add this file to `.gitignore`

File: cypress/support/commands.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```
Comment (sirLisko) [inline cypress/support/commands.js:1]: I think we can delete `cypress/support` folder 🤔

**Review by sirLisko — COMMENTED** [review]
Tests are looking good, just two small comments :)

**Comment by LBHMVillalobos** [comment]
I've deleted the unused files. It should be ok now @sirLisko. Thanks!

**Review by sirLisko — APPROVED** [review]
Code looks good 👍 
Tested and it works as expected 🚀
Good one! 👏

**Pull request merged**